### PR TITLE
AIM-857 Add support for page funnels.

### PR DESCRIPTION
**Pull request by duffj**

### CODE REVIEW

**Comment by duffj** [comment]
@anuragkapur / @mclarke47 / @frj / @tjcutajar - please review :smile_cat:

File: client/src/main/java/com/ft/membership/spoorclient/SpoorParameters.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
     static {
         mapper.registerModule(new Jdk8Module());
         mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        mapper.setPropertyNamingStrategy(PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
```
Comment (tjcutajar) [inline client/src/main/java/com/ft/membership/spoorclient/SpoorParameters.java]: Is this desirable/will this break other things?

Comment (duffj) [inline client/src/main/java/com/ft/membership/spoorclient/SpoorParameters.java]: Good question; it seems like the entire spoor API uses lowercase/underscores for its API; if it deviates from that standard then we should probably be hitting them over the head with a big standards stick.

**Comment by duffj** [comment]
@mclarke47 - potentially yes, but we're never actually consuming data from spoor, just sending it, so I don't think it's a high priority until we start requesting data from spoor. Would you agree?

**Comment by mclarke47** [comment]
@duffj sure

**Pull request merged**